src/stretch/agent/operations/pregrasp.py

```diff
@@ -32,11 +32,10 @@ def can_start(self):
             return False
 
         start = self.robot.get_base_pose()
-        if not self.navigation_space.is_valid(start):
+        if not self.navigation_space.is_valid(start, verbose=True):
             self.error(
                 f"{self.name}: [ERROR]: Robot is in an invalid configuration. It is probably too close to geometry, or localization has failed."
             )
-            breakpoint()
```

Comment:
What happens if there's no break?

Reply:
This change was made before I added the new filter. Reverted.